web-services/query/src/main/java/datawave/webservice/query/configuration/GenericQueryConfiguration.java

```diff
@@ -137,15 +149,19 @@ public Long getMaxQueryResults() {
     }
     
     public void setMaxQueryResults(Long maxQueryResults) {
-        this.maxQueryResults = maxQueryResults <= 0l ? Long.MAX_VALUE : maxQueryResults;
```

Comment:
What's wrong with the ternary operator?

Reply:
Nothing, it was a personal preference. This line won't be changed once I rebase to pick up the maxResults changes.